swagger-def/internal.yaml

```
@@ -1794,6 +1794,17 @@ definitions:
         type: integer
         example: 7000
         x-nullable: true
+      entitlement:
+        type: object
+        properties:
+          proGear:
+            type: integer
+            example: 2000
+            x-nullable: true
+          proGearSpouse:
+            type: integer
+            example: 500
+            x-nullable: true
```

If you want to just define this in this current file you can likely just do the following
```suggestion
        $ref: '#/definitions/Entitlement'
```
You may also need to updated the orders.go file to use `internalmessages.Entitlement` instead of `internalmessages.OrdersEntitlement` once you make this change.

We can also get rid of the Entitlement.yaml file as well. Alternatively, if you want to define a new file you can add the definition there and reference it similar to what we do for ppmShipment([ref](https://github.com/transcom/mymove/blob/master/swagger-def/internal.yaml#L2236-L2237)) and the associated `PPMShipment.yaml` file.

---

The [Entitlement.yaml](https://github.com/transcom/mymove/pull/9480/files#diff-054742b2e8d142980849dc48ad9d5ea65acf602c47aa5c507ab7a369dd987ce2) file looks empty so let's delete that before merging.